src/components/input/IssueFilterDropdowns.vue

```diff
@@ -113,52 +113,58 @@ const props = defineProps({
     },
     trackableId: {
         type: String,
-        required: true
+        required: false
     },
     stateIndices: {
         type: Array as PropType<number[]>,
         required: true
+    },
+    onlyAssigned: {
+        type: Boolean,
+        required: false,
+        default: false
     }
 });
 
 const client = useClient();
 const store = useAppStore();
+const userId = computed(() => store.user?.id);
 
 const templateIds = useFilterOption("template");
 const templateInput = computed(() => {
     return templateIds.value.length > 0 ? { id: { in: templateIds.value } } : undefined;
 });
-const templateFetch = async (search: string) =>
+const templateFetch = async (search: string) => props.trackableId ?
```

Comment:
as it seems like I missed this last time: is there any error handling if any of these fetches fail?